locale/sl_SI/manager.xml

```diff
@@ -0,0 +1,362 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/en_US/manager.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University Library
```

Comment:
The copyright statements should all end in 2018, and `Simon Fraser University Library` should be changed to `Simon Fraser University`.